nbextensions/bulkImportCell/main.js

```diff
@@ -53,14 +63,44 @@ define([
         }));
     }
 
+
+    function setupWorkspace(workspaceUrl) {
```

Comment:
borrowed from appCell

Reply:
This seems like a good method to move to some utility module. I bet it actually exists somewhere already, I'll look.

Reply:
good call - i'll add this to the todo list for later